while-true-do/ansible-galaxy-skeleton
# Update: Links properly set

- Update: docs, meta, tests
- Update: links no longer using blog
- Update: CODE_OF_CONDUCT included

Resolves: #24, #25
